### DISCUSSION TITLE
[Question] How to Automatically Show Horizontal Scrollbar at Bottom of Page in Flet Web Application?

### DISCUSSION THREAD

**Discussion by HuaxiaGoodBoy** [flet-dev/flet#4078] [Q&A]
### Question

I am developing a web application using the Flet library in Python. I want to add both horizontal and vertical scrollbars to a page, with the horizontal scrollbar only appearing when the user scrolls to the bottom of the page. How can I automatically show the horizontal scrollbar when the mouse is at the bottom of the page?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
How is you UI structured? Will be nice if you could share runnable code.
